source/guides/1.1/getting-started.md

```diff
@@ -435,7 +435,7 @@ We can verify it all works as expected with a unit test:
 
 ```ruby
 # spec/bookshelf/entities/book_spec.rb
-require 'spec_helper'
+require_relative '../../spec_helper'
```

Comment:
I noticed that the generators generate the tests with a `require_relative`. I think either the generator or the guide should be adjusted.